getsolus/packages
# cinelerra-gg: Update to 5.1.2024.02.29

**Summary**

- Release notes are available [here](https://cinelerra-gg.org/download/releasenotes.pdf)
- Add appstream metainfo (Part of #1389)

**Test Plan**

- Create simple video
- Verify metainfo with `appstream-builder --packages-dir=. --include-failed -v`

**Checklist**
